apps/judicial-system/backend/src/app/modules/subpoena/subpoena.service.ts

```diff
@@ -320,4 +334,44 @@ export class SubpoenaService {
       return { delivered: false }
     }
   }
+
+  async getSubpoena(subpoena: Subpoena, user?: TUser): Promise<Subpoena> {
```

Comment:
When retrieving the subpoena the subpoena info is refreshed from the police system if it has been delivered to the police (subpoenaId is not null), but does not have a service status yet.